restapi.go

```diff
@@ -1335,6 +1335,30 @@ func (s *Session) ChannelMessageSendEmbed(channelID string, embed *MessageEmbed)
 	return
```

Comment:
Could you change this method to:

return s.ChannelSendEmbedWithMessage(channelID, "", embed)

seems like that should work.